### ISSUE TITLE
!! Documentation is wrong, please add XS* css directive

### ISSUE THREAD

**Issue by OskHa** [angular/material#5977]
I have a very strange problem.

We are installing angular assets using bower,

on development machine, both on host (OS X,) and guest (debian), is the produced css from 'angular-material.scss' are totally fine and working great.

On production machine (also debian)  gulp is compiling all files properly BUT with a very small difference. All sm* definitions are wrong.

Dev:
```
@media (max-width: 767px)
.flex-sm-10, .layout-row>.flex-sm-10, .layout-sm-row>.flex-sm-10
```

Production:
```
@media (max-width: 959px) and (min-width: 600px)
.flex-sm-10, .layout-row>.flex-sm-10, .layout-sm-row>.flex-sm-10 
```

As you see the @media directive is completely wrong.

We are using on all machines and all environments same setup:

```
node -v
v5.1.0

nodejs -v
v5.1.0

npm -v
3.3.12

bower -v
1.6.8

bower list
├─┬ angular-material#a834294239
│ ├── angular#1.4.8 (1.5.0-build.4406+sha.7ffb2d3 available)
│ ├─┬ angular-animate#1.4.8 (1.5.0-build.4406+sha.7ffb2d3 available)
│ │ └── angular#1.4.8 (latest is 1.5.0-build.4406+sha.7ffb2d3)
│ └─┬ angular-aria#1.4.8 (1.5.0-build.4406+sha.7ffb2d3 available)
│   └── angular#1.4.8
├─┬ angular-messages#1.4.8 (latest is 1.5.0-build.4406+sha.7ffb2d3)
│ └── angular#1.4.8
├── components-font-awesome#069663a841
└── jquery#2.1.4 (latest is 3.0.0-alpha1+compat)
```

Any help will be very appreciated. I already tried anything to find the issue...

**Comment by OskHa** [comment]
Problem is solved. HEAD just changed some SCSS directives, the smaller view is XS* not SM*.
It would be great if someone could update the documentation.

**Comment by ThomasBurleson** [comment]
Fixed with commit SHA 2413be43f71.
